apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -505,6 +505,7 @@ type DeprecatedSystemS3Spec struct {
 
 type SystemS3Spec struct {
 	ConfigurationSecretRef v1.LocalObjectReference `json:"configurationSecretRef"`
+	S3STSEnabled           bool                    `json:"s3StsEnabledSts,omitempty"`
```

Comment:
This should be renamed as suggested above by @eguzki

Reply:
done